sphinx/builders/texinfo.py

```diff
@@ -70,7 +70,7 @@ def get_relative_uri(self, from_: str, to: str, typ: str | None = None) -> str:
         # ignore source path
         return self.get_target_uri(to, typ)
 
-    def init_document_data(self) -> None:
```

Comment:
Could we keep this method just in case for backward compatibility but deprecate it (it's not documented but it's also not underscored...)